src/main/resources/application.conf

```diff
@@ -199,32 +199,28 @@ mantis {
   }
 
   consensus {
-    # FIXME this was moved here from the `mining` section
     active-timeout = 5 seconds
 
     # New timeout to replace `ommer-pool-query-timeout` for uses outside ethash mining
     get-transaction-from-pool-timeout = 5.seconds
```

Comment:
same as above, this has nothing to do with consensus. How about moving it to `txPool`?

Reply:
Why do you say so? It is a timeout specified and needed by a client of the pool, not the pool itself. It is currently used in both the ethereum miner and the raft "miner". So it seems to be precisely about consensus, a desing that pre-existed, I did not introduce it.

Also, in the pre-pluggable-consensus setup, the ethereum `Miner` used `ommer-pool-query-timeout` for the same reason inside the `mining` section (so in a consensus-related section). I just introduced another timeout for better reflecting the intent, since `ommer-pool-query-timeout` was properly used elsewhere (but again in the `Miner`).

Reply:
To be honest it's really just a boilerplate needed because we use the ask pattern. I, personally, would just make it a constant in code. My point was, it has nothing to do with consensus because it's strictly an implementation detail.

Reply:
[Moved](https://github.com/input-output-hk/mantis/pull/410/commits/410386a3cd1878595e4ed95ce245d0a30c18a482) `get-transaction-from-pool-timeout` to `txPool`.